src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -78,18 +79,21 @@ public void run() throws Exception {
              *      2. has a Closure parameter in one of the arguments, not in the receiver
              */
             Predicate<ExecutableElement> selector = (e) -> {
-                return e.getKind() == ElementKind.METHOD
-                    && e.getModifiers().containsAll(PUBLIC_STATIC)
-                    && e.getParameters().subList(1, e.getParameters().size()).stream()
-                        .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
-                    && !EXCLUSIONS.contains(e.getEnclosingElement().getSimpleName().toString()+"."+e.getSimpleName().toString());
+                boolean r = e.getKind() == ElementKind.METHOD
+                        && e.getModifiers().containsAll(PUBLIC_STATIC)
```

Comment:
Mm, OK, need to translate helper methods too.

Another idea was to simply copy `DefaultGroovyMethods.java` to `CpsDefaultGroovyMethods.groovy` (after only `s/DefaultGroovyMethods/CpsDefaultGroovyMethods/g`) and then compile it using `CpsTransformer` to bytecode. That would let us use the inner classes etc. and avoid most of the mechanics of translation here. But anyway it would not help with the main issue facing me right now, which is the fact that Java static method overload resolution does not survive CPS transformation.